sandbox/experimental/data-packages/dwc-dp/0.1/index.json

```diff
@@ -186,13 +200,6 @@
             "title": "GeneticSequenceMedia",
```

Comment:
I suggest to remove Media here and in the description. Simply state that this is the GeneticSequence.

Reply:
I disagree. This is a join table between GeneticSequence and Media, so that GeneticSequences can appear in Media. It follows the pattern of all [Class]Media join tables.

Reply:
You are correct. It's the github view that has hidden lines, making it appear under the http://rs.gbif.org/sandbox/experimental/data-packages/dwc-dp/0.1/table-schemas/genetic-sequence.json URL